instrumentation/opentelemetry-instrumentation-aiohttp-client/setup.py

```diff
@@ -18,20 +18,42 @@
 
 
 import os
+from configparser import ConfigParser
 
 import setuptools
 
+config = ConfigParser()
+config.read("setup.cfg")
+
+# We provide extras_require parameter to setuptools.setup later which
+# overwrites the extra_require section from setup.cfg. To support extra_require
+# secion in setup.cfg, we load it here and merge it with the extra_require param.
+extras_require = {}
+if "options.extras_require" in config:
+    for key, value in config["options.extras_require"].items():
+        extras_require[key] = [v for v in value.split("\n") if v.strip()]
+
 BASE_DIR = os.path.dirname(__file__)
-VERSION_FILENAME = os.path.join(
+PACKAGE_FILENAME = os.path.join(
     BASE_DIR,
     "src",
     "opentelemetry",
     "instrumentation",
     "aiohttp_client",
-    "version.py",
+    "package.py",
```

Comment:
will renaming the `version.py` files break the release process? see `eachdist.py`: https://github.com/open-telemetry/opentelemetry-python-contrib/blob/v0.20b0/scripts/eachdist.py#L578

Reply:
Thanks. Will fix.